components/mediators/cache/org.wso2.carbon.mediator.cache/src/main/java/org/wso2/carbon/mediator/cache/MediatorCacheInvalidatorMBean.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.mediator.cache;
+
+/**
+ * Cache invalidator mbean interface
+ */
+public interface MediatorCacheInvalidatorMBean {
+
+	/**
+	 * This abstract method should be implemented to invalidate the total mediator Cache.
+	 */
+	public void invalidateMediatorCache();
```

Comment:
Methods declared in an interface are always public and abstract. Therefore, having the access modifier can sometimes be considered redundant.